apps/web/src/components/sidebar/IndexingStatus/index.tsx

```diff
@@ -16,6 +16,7 @@ const useIndexingStatus = () => {
 
   return useAsync(
     () => {
+      if (count === undefined) return
```

Comment:
How can this case happen? Afaik the default value is 0 so it can never be undefined.

Reply:
I added it only to satisfy the linter as it's a dependency.